scenarios/reproducers/3-nodes.yml

```diff
@@ -16,6 +16,11 @@ cifmw_libvirt_manager_configuration:
   vms:
     compute:
       amount: 1
+      root_part_id: >-
+        {{
+          (cifmw_repo_setup_os_release is defined and cifmw_repo_setup_os_release == 'rhel') |
+          ternary(4, 1)
```

Comment:
ack seems consistent with what we have in the validated-arch 1 https://github.com/marios/ci-framework/blob/37fd6398b31860e16cc30c82d9e9a2486b5c6e6d/scenarios/reproducers/validated-architecture-1.yml#L45 

is this something that will depend on the actual setup we run it on though? I am guessing the above works for the systems we are testing on but in theory the partition system used by the target machines may vary?

Just wondering if we need some note about this unless we already mention that in the docs?

Reply:
it's not the target hypervisor, this is really the virtual machine drive (the .qcow2). I don't really expect it to change, sooo... That parameter is in the doc iirc.